Dockerfile

```diff
@@ -7,9 +7,14 @@ RUN go mod download
 
 COPY pkg ./pkg
 
+RUN curl -o /sloop/aws-iam-authenticator https://amazon-eks.s3-us-west-2.amazonaws.com/1.14.6/2019-08-22/bin/linux/amd64/aws-iam-authenticator \
```

Comment:
Why is this needed in the dockerfile?  In the bash script below you are not calling it.  And I am able to connect to AWS without it from my workstation.

Reply:
When I tried Sloop docker without this change, I got the following error:
```
E1118 07:21:46.316905       1 reflector.go:153] pkg/mod/k8s.io/client-go@v0.0.0-20191003035859-a746c2f219b7/tools/cache/reflector.go:105: Failed to list *v1.Pod: Get https://F5483839EA2647AE729EDBFE036C2559.gr7.us-west-2.eks.amazonaws.com/api/v1/pods?limit=500&resourceVersion=0: getting credentials: exec: exec: "aws-iam-authenticator": executable file not found in $PATH
```
So k8s.io/client-go has a dependency on it.  I don't have to call it explicitly.

Reply:
My mistake, I tried it bare metal and it worked but I had this installed.